src/components/pages/admin/PageProductImport/components/CSVFileImport.tsx

```diff
@@ -43,8 +43,11 @@ export default function CSVFileImport({url, title}: CSVFileImportProps) {
     console.log('Uploading to: ', response.data)
     const result = await fetch(response.data, {
         method: 'PUT',
-        body: file
-      })
+        body: file,
+        headers: {
+          'Content-Type': 'text/csv',
```

Comment:
You are trying to set `'Content-Type': 'text/csv'` for ALL types of selected files. I tried to upload the PDF file and there were no errors because you send text/csv headers for all files. It would be better to get file mime type (`file.type` will return it). The signed URL will automatically return 403 error if the file format is wrong.

Reply:
agree with this comment!